pandas/tests/frame/indexing/test_where.py

```diff
@@ -705,6 +707,11 @@ def test_where_categorical_filtering(self):
 
         tm.assert_equal(result, expected)
 
+    @pytest.mark.xfail(
+        not np_version_gte1p24,
+        reason="Changed NumPy behavior for >1D non-tuple sequence indexing",
+        strict=False,
```

Comment:
is this flaky?

Reply:
Yea I couldn't reproduce locally but saw it on CI. Next push I'll take this off and we can see if it shows back up

Reply:
Here's the error this creates on the min versions CI

https://github.com/pandas-dev/pandas/actions/runs/6200248114/job/16834535267?pr=54506#step:8:181